sys-devel/ucpp/ucpp-1.3.5.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A C preprocessor compliant to ISO-C99"
+HOMEPAGE="https://gitlab.com/scarabeusiv/ucpp"
+SRC_URI="https://gitlab.com/scarabeusiv/${PN}/uploads/79f08e39c676f15ed8a59335f6c9b924/${P}.tar.xz"
```

Comment:
`SRC_URI="https://gitlab.com/scarabeusiv/ucpp/-/archive/${PV}/ucpp-${PV}.tar.bz2"`

Reply:
That's the URI for the source archive, not the distribution tarball. The latest distribution tarball releases are located here: https://gitlab.com/scarabeusiv/ucpp/-/releases